lib/standard/string.nit

```diff
@@ -47,6 +47,11 @@ abstract class Text
 	#     assert "".length == 0
 	fun length: Int is abstract
 
+	# Length in bytes of the string
+	#
+	#    assert "12345".bytelen == 5
```

Comment:
Yup, but right now I can't really since length is equal to bytelen until we replace Byte-oriented services by Char-oriented services and regen c_src.
Until that, I can't modify this since I'll break the bootstrap.
But when it's done, sure, any foreign language will show the difference between length and bytelen !